python/cudf/cudf/utils/dtypes.py

```diff
@@ -118,6 +118,10 @@ def is_numerical_dtype(obj):
     )
 
 
+def is_range_compatible_dtype(dtype):
+    return np.issubdtype(dtype, np.signedinteger)
```

Comment:
Is it necessary to abstract this?

Reply:
Probably not. At this point, I don't see when other method may reuse this method.